src/migrations/20210215181557-convert-string-types-to-text.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('ActivityReports', 'additionalNotes', { type: Sequelize.TEXT });
+    await queryInterface.changeColumn('ActivityReports', 'context', { type: Sequelize.TEXT });
```

Comment:
Can we wrap these into Promise.all ? This is to avoid a partial successful completion. Alternatively, we could split the changes into separate migrations.